00_inspect_raws.py

```diff
@@ -0,0 +1,30 @@
+"""Inspect raw files manually."""
+# %%
+# imports
+import mne
+
+from config import FPATH_DS
+from utils import get_raw_data
+
+# %matplotlib #toggle interactive mode
```

Comment:
what does this command do? Is that a command or just a comment?

I always do `%matplotlib qt` (interactive window) or `%matplotlib inline` (plots in console)